test/framework/flux.go

```diff
@@ -755,6 +755,11 @@ func (e *ClusterE2ETest) validateWorkerNodeMachineSpec(ctx context.Context, clus
 				e.T.Logf("Waiting for WorkerNode Specs to match - %s", err.Error())
 				return err
 			}
+			if !reflect.DeepEqual(cloudstackWorkerConfig.Spec.Symlinks, csMachineTemplate.Spec.Spec.Spec.Symlinks) {
```

Comment:
could you add symlinks to some e2e test, or make a new test case maybe?

Reply:
I added new test cases for symlinks. 

Adding e2e is a good idea, we need to do that disk offering and symlinks, I will add it in a separate PR.